Solution1/ConsoleApp/AutoMap/WeatherProfile.cs

```diff
@@ -0,0 +1,37 @@
+﻿using DataAccessLayer.Model;
+using BusinessLayer.DTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConsoleApp.AutoMap
+{
+    public class WeatherProfile : AutoMapper.Profile 
+    {
+        public WeatherProfile()
+        {
+            CreateMap<Weather, WeatherDTO>()
+                .ForMember(dest => dest.CityName, conf => conf.MapFrom(src => src.CityName))
```

Comment:
CityName and CityName - mapper is created for automatic resolving properties with same name

Reply:
+